### DISCUSSION TITLE
How to get the content of an external txt file?

### DISCUSSION THREAD

**Discussion by nalcakar** [pixijs/pixijs#8917] [Q&A]
Hi,
I am preparing a sentence game for learners of English. I have a txt file containing sentences like "I am a student.*She is my aunt.*They are not my friends.*Go home and rest for a while."

I need to get these sentences into the game using something like ajax ...

So, I wonder if there is a way of getting the content an external txt file into the pixijs. app? thanks

**Comment by domis86** [comment]
You mean something like this? https://stackoverflow.com/questions/196498/how-do-i-load-the-contents-of-a-text-file-into-a-javascript-variable
